tardis/montecarlo/montecarlo_configuration.py

```diff
@@ -1,18 +1,74 @@
+from astropy import units as u
 from tardis import constants as const
+from tardis.montecarlo import (
+    montecarlo_configuration as montecarlo_configuration,
+)
+from tardis.montecarlo.montecarlo_numba.numba_interface import (
+    LineInteractionType,
+)
+
+ENABLE_FULL_RELATIVITY = False
+TEMPORARY_V_PACKET_BINS = 0
+NUMBER_OF_VPACKETS = 0
+MONTECARLO_SEED = 0
+LINE_INTERACTION_TYPE = None
+PACKET_SEEDS = []
+DISABLE_ELECTRON_SCATTERING = False
+DISABLE_LINE_SCATTERING = False
+SURVIVAL_PROBABILITY = 0.0
+TAU_RUSSIAN = 10.0
```

Comment:
I would consider adding either a small comment or proper documentation as to what each config constant does. E.g. ``MONTECARLO_SEED`` is self-explanatory, but I would have no idea what ``TAU_RUSSIAN`` means or does. In case there already exists such documentation and i just missed it, maybe add a comment above the constants linking to it.

Reply:
These constants will almost certainly be broken out into a proper documented configuration with defaults